backend/core/auth.py

```
@@ -0,0 +1,138 @@
+from fastapi import Depends, HTTPException, Security, status
+from fastapi.security import HTTPBearer, HTTPAuthorizationCredentials
+from typing import List, Dict, Any
+from backend.db.supabase_client import supabase_client
+from backend.core.config import settings
```

Ruff (0.11.9)

`backend.core.config.settings` imported but unused

Remove unused import: `backend.core.config.settings`

(F401)